src/SLCore/Listener/Analysis/RaiseFindingToAnalysisIssueConverter.cs

```diff
@@ -82,5 +100,14 @@ private static IAnalysisIssueFlow[] GetFlows(List<IssueFlowDto> issueFlows)
         private static IEdit GetEdit(TextEditDto textEdit) =>
             new Edit(textEdit.newText,
                 new TextRange(textEdit.range.startLine, textEdit.range.endLine, textEdit.range.startLineOffset, textEdit.range.endLineOffset, null));
+
+        private static HotspotPriority? GetHotspotPriority(RaisedHotspotDto hotspotDto)
+        {
+            if(hotspotDto.vulnerabilityProbability == null)
+            {
+                return null;
+            }
+            return (HotspotPriority)Enum.Parse(typeof(HotspotPriority), hotspotDto.vulnerabilityProbability.ToString(), ignoreCase:true);
```

Comment:
convert to switch-case & add it to ModelConversionExtensions

Reply:
I agree to move it to ModelConversionExtensions.

But why a manual switch is better than a Parse converter? Using Parse has the advantage that any new value will be automatically supported. And if the value is not found in the expected enum, it will throw an exception

Reply:
Summary:
* Not any new value, they have to match. If there would be such case, the method will need to be rewritten...
* But it won't happen since hotspots are pretty much deprecated now. So the declarative switch approach has the advantage of being explicit and clear.
* This will make sure that the decoupling of dto and model is respected
* Does not introduce intermediary data structure